### PR TITLE
feat(prettier): prettify .ts files

### PR DESCRIPTION
**Pull request by christianmemije**
## Description

prettify .ts files

#### Test Steps

- [ ] go through every file and make sure everything looks good
- [x] `npm i`
- [x] `npm serve`still works
- [x] `npm run serve:prod` still works.
- [x] `npm run tslint` passes.
- [x] `npm test` passes and code coverage is not lower.
- [x] `npm run build:lib` still works.

### CODE REVIEW

File: src/app/app.animations.ts [hunk 1]
```diff
@@ -1,58 +1,74 @@
-import { animate, AnimationTriggerMetadata, state, style, transition, trigger, group, animateChild, query } from '@angular/animations';
+import {
```
Comment (richavyas) [inline src/app/app.animations.ts]: Does `prettier` also take care of sorting the imports alphabetically? what's the practice around that?

Comment (christianmemije) [reply src/app/app.animations.ts]: Nope it does not sort imports. That would be something that can be done with tslint.

File: src/platform/core/common/pipes/bytes/bytes.pipe.spec.ts [hunk 2]
```diff
@@ -25,8 +25,8 @@ describe('TdBytesPipe', () => {
     expect(pipe.transform(10000000, undefined)).toEqual('9.54 MiB');
     expect(pipe.transform(10485760, undefined)).toEqual('10 MiB');
     expect(pipe.transform(10201000, 3)).toEqual('9.728 MiB');
-    expect(pipe.transform(3.81861e+10, undefined)).toEqual('35.56 GiB');
-    expect(pipe.transform(1.890381861e+14, undefined)).toEqual('171.93 TiB');
-    expect(pipe.transform(5.35765e+16, undefined)).toEqual('47.59 PiB');
+    expect(pipe.transform(3.81861e10, undefined)).toEqual('35.56 GiB');
+    expect(pipe.transform(1.890381861e14, undefined)).toEqual('171.93 TiB');
```
Comment (richavyas) [inline src/platform/core/common/pipes/bytes/bytes.pipe.spec.ts]: symbols `+` has been removed from the pipe input. this doesn't seem right.

File: src/platform/core/common/pipes/bytes/bytes.pipe.spec.ts [hunk 3]
```diff
@@ -25,8 +25,8 @@ describe('TdBytesPipe', () => {
     expect(pipe.transform(10000000, undefined)).toEqual('9.54 MiB');
     expect(pipe.transform(10485760, undefined)).toEqual('10 MiB');
     expect(pipe.transform(10201000, 3)).toEqual('9.728 MiB');
-    expect(pipe.transform(3.81861e+10, undefined)).toEqual('35.56 GiB');
-    expect(pipe.transform(1.890381861e+14, undefined)).toEqual('171.93 TiB');
-    expect(pipe.transform(5.35765e+16, undefined)).toEqual('47.59 PiB');
+    expect(pipe.transform(3.81861e10, undefined)).toEqual('35.56 GiB');
```
Comment (richavyas) [inline src/platform/core/common/pipes/bytes/bytes.pipe.spec.ts]: symbols + has been removed from the pipe input. this doesn't seem right.

Comment (christianmemije) [reply src/platform/core/common/pipes/bytes/bytes.pipe.spec.ts]: They are equivalent.

**Pull request merged**